src/xrpld/app/tx/detail/MPTokenIssuanceSet.cpp

```diff
@@ -180,9 +267,48 @@ MPTokenIssuanceSet::doApply()
     else if (txFlags & tfMPTUnlock)
         flagsOut &= ~lsfMPTLocked;
 
+    if (auto const mutableFlags = ctx_.tx[~sfMutableFlags])
+    {
+        if (*mutableFlags & tfMPTSetCanLock)
+            flagsOut |= lsfMPTCanLock;
+        else if (*mutableFlags & tfMPTClearCanLock)
+            flagsOut &= ~lsfMPTCanLock;
+
+        if (*mutableFlags & tfMPTSetRequireAuth)
+            flagsOut |= lsfMPTRequireAuth;
+        else if (*mutableFlags & tfMPTClearRequireAuth)
+            flagsOut &= ~lsfMPTRequireAuth;
+
+        if (*mutableFlags & tfMPTSetCanEscrow)
+            flagsOut |= lsfMPTCanEscrow;
+        else if (*mutableFlags & tfMPTClearCanEscrow)
+            flagsOut &= ~lsfMPTCanEscrow;
+
+        if (*mutableFlags & tfMPTSetCanTrade)
+            flagsOut |= lsfMPTCanTrade;
+        else if (*mutableFlags & tfMPTClearCanTrade)
+            flagsOut &= ~lsfMPTCanTrade;
+
+        if (*mutableFlags & tfMPTSetCanTransfer)
+            flagsOut |= lsfMPTCanTransfer;
+        else if (*mutableFlags & tfMPTClearCanTransfer)
+            flagsOut &= ~lsfMPTCanTransfer;
```

Comment:
in this case, we actually would need to clear out the TransferFee field on the MPTokenIssuance. Would need to add a unit test for this as well